### DISCUSSION TITLE
makeiso.sh does not install wget for its own use

### DISCUSSION THREAD

**Discussion by tadashi-oya** [telekom-security/tpotce#1059] [General]
makeiso.sh does not install wget for its own use.
https://github.com/tadashi-oya/tpotce/commit/e670a93bf37661f4641a422ac20a5c958bd39256

**Comment by t3chn0m4g3** [comment]
Thanks, fixed with ab46fcb298d9623fba491d2b5f97319b7ac8c3d2
